pom.xml

```diff
@@ -483,7 +483,7 @@
         <version.org.jboss.jboss-transaction-spi>8.0.0.Final</version.org.jboss.jboss-transaction-spi>
         <version.org.jboss.metadata>16.0.0.Final</version.org.jboss.metadata>
         <version.org.jboss.mod_cluster>2.1.0.Final</version.org.jboss.mod_cluster>
-        <version.org.jboss.narayana>7.1.0.Final</version.org.jboss.narayana>
+        <version.org.jboss.narayana>7.2.0.Final</version.org.jboss.narayana>
```

Comment:
com.arjuna.common.internal.util.ClassloadingUtility.loadAndInstantiateClass added a call to Class.getDeclaredConstructor which fails, at least in some situations, when the SecurityManager is enabled, leading to the test failures.

The correct fix is tricky as that looks to be a general utility class, so adding a privileged block inside it is likely wrong. More likely such blocks belong in whatever callers need it and are safe to add it.

Reply:
I see, thanks @bstansberry.  I wonder if that is a problem related to 7.2.0.Final release though. I will have a look at that and fix it for 7.0.3.Final release.